advocacy_docs/supported-open-source/patroni/03-rhel8_quick_start.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: 'Quick Start (RHEL 8)'
+description: "A walkthrough: setup a demo cluster using Patroni and etcd"
+---
+
+### Description
+
+This section walks you through the various steps to create an highly available **demo** cluster (using **PostgreSQL** version 14 on Rocky Linux 8) where the etcd hosts will be called **etcd1**, **etcd2** and **etcd3**, and the 2 PostgreSQL nodes in _Streaming Replication_: **pg-patroni1** and **pg-patroni2**.
+
+![Demo diagram](images/etcd-demo.png)
+
+---
+
+### Etcd
+
+See the [Etcd installation guide](98-etcd/) to install and setup etcd.
+
+### PostgreSQL
+
+On both **pg-patroni1** and **pg-patroni2** hosts, install PostgreSQL:
+
+```bash
+$ sudo dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+$ sudo dnf -qy module disable postgresql
+$ sudo dnf install -y postgresql14-server postgresql14-contrib
+$ sudo systemctl disable postgresql-14
+```
+
+Patroni will bootstrap (create) the initial PostgreSQL cluster and be in charge of starting the service.
```

Comment:
Probably a good idea to add "so be sure systemctl is disabled for the postgres service" or something around those words. Patroni will not even start if systemctl has started the process

Reply:
adding _, so be sure `systemctl` is disable for the PostgreSQL service._

Reply:
fixed in 5bf5f28